File: src/lib/android/libbase/override/properties.cc
// Copyright 2023 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include <android-base/properties.h>

namespace android {
namespace base {

std::string GetProperty(const std::string& key, const std::string& default_value) {
  return default_value;
}

bool GetBoolProperty(const std::string& key, bool default_value) { return default_value; }

}  // namespace base
}  // namespace android

File: src/tee/manager/tests/src/properties.rs
// Copyright 2024 The Fuchsia Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

use anyhow::Error;
use std::path::Path;
use tee_properties::{PropSet, PropSetType};

#[fuchsia::test]
async fn read_system_properties_from_ta_manager() -> Result<(), Error> {
    // The /properties dir is expected to be routed from TA manager.
    let res = PropSet::from_config_file(
        Path::new("/properties/system_properties"),
        PropSetType::TeeImplementation,
    );
    assert!(res.is_ok(), "Failed to load system properties: {:?}.", res.err());
    Ok(())
}
